### DISCUSSION TITLE
Directional Spot on the screen in browser + text based graphics

### DISCUSSION THREAD

**Discussion by LovesPictures** [mozilla/hubs#2839] [General Questions & Discussion] [answered]
Hi 

Is there any way to remove the directional spot which appears when looking at a hubs space in the browser? I would like to record a walkthrough, ideally without it.

Any hints on putting text-based graphics into a hubs space too please.
Thanks

**Comment by emclaren** [comment]
Hi there - apologies for the delay, this discussion topic wasn't coming up in our list of "unanswered" questions.

When you refer to the "directional spot" - is it this?
<img width="165" alt="Screen Shot 2020-09-25 at 3 41 55 PM" src="https://user-images.githubusercontent.com/6835324/94321866-b869f700-ff45-11ea-8b72-81ac8876bf63.png">

If so, there isn't a built-in way to change the size, but for a walkthrough you can hide it by making a change in your browser's developers tools. In Firefox you would do this by opening up the Inspector menu
<img width="492" alt="Screen Shot 2020-09-25 at 3 47 24 PM" src="https://user-images.githubusercontent.com/6835324/94322125-768d8080-ff46-11ea-9b72-388a9fce19f1.png">

Expand the HTML sections until you find <a-sphere id="right-cursor" ...

<img width="1103" alt="Screen Shot 2020-09-25 at 3 45 44 PM" src="https://user-images.githubusercontent.com/6835324/94322134-7db48e80-ff46-11ea-872c-d0a03ca0156f.png">

Find where it says radius = .02, and change it to something smaller. Once you click away in from the text you were editing it should apply the change in your browser window - your cursor should now be so small its invisible. If you make a mistake here, don't worry - the console lets you play with the code, but you aren't making any real changes, if you refresh the page everything will be reset.

Note, if you are interested we have some tips on recording in Hubs here: https://blog.mozvr.com/recording-inside-of-hubs/  (like pressing the ~ key hides the interface)

**Comment by emclaren** [comment, marked as answer]
**To respond to your question about Text Graphics**

For text graphics you can create them in-world by typing text into the chatbox and then pressing the magic wand in the chatbox (note this user interface might be updated somewhat in the coming months)
<img width="621" alt="Screen Shot 2020-09-25 at 3 31 34 PM" src="https://user-images.githubusercontent.com/6835324/94321439-62488400-ff44-11ea-8e2e-d83800acc422.png">

This will create text chat bubbles that look like this:
<img width="604" alt="Screen Shot 2020-09-25 at 3 31 42 PM" src="https://user-images.githubusercontent.com/6835324/94321449-670d3800-ff44-11ea-84b0-986da2f6425d.png">

Another technique for creating text-based graphics is to create them using a 2D image editing tool (gimp/photoshop/canva.com etc.) and then importing your creations as image files into Hubs or Spoke - for example:

<img width="668" alt="Screen Shot 2020-09-25 at 3 35 57 PM" src="https://user-images.githubusercontent.com/6835324/94321573-dbe07200-ff44-11ea-8351-9a99410813da.png">

We also just released a feature called "media frames". These are spots you can place inside of your Hubs scene using Spoke, and then when you bring graphics/3D objects near that space it automatically pops them into place and enlarges them to fill (see screenshot of a video being placed in the blue media frame below). This could be a helpful tool if you want to be replacing any text based graphics in your room regularly.  
![93437200-8b9c4c80-f880-11ea-9849-7bbe80e39dc9](https://user-images.githubusercontent.com/6835324/94321770-6fb23e00-ff45-11ea-8875-faa8b550e069.png)
